### PR TITLE
added 3 libs: concorde_tsp_solver, dlib, and opengm

### PR DESCRIPTION
**Pull request by ipa-rmb**

### CODE REVIEW

File: libconcorde_tsp_solver/common/files/TSPlib_file.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+NAME: ipa-building-navigation
```
Comment (floweisshardt) [inline libconcorde_tsp_solver/common/files/TSPlib_file.txt]: please rename "ipa-building-navigation"

File: libconcorde_tsp_solver/package.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<package>
+	<name>libconcorde_tsp_solver</name>
+	<version>0.1.0</version>
+	<description>Wrapper for the concorde traveling salesman problem solver from http://www.math.uwaterloo.ca/tsp/concorde/downloads/downloads.htm</description>
+
+	<author email="florian.jordan@ipa.fraunhofer.de">Florian Jordan</author>
+	<maintainer email="rmb@ipa.fhg.de">Richard Bormann</maintainer>
+
+	<license>see original software</license>
```
Comment (floweisshardt) [inline libconcorde_tsp_solver/package.xml]: what is the license of the original software? 

anyway, the license we list here is just for our ROS wrapper. We should use LGPL here.

File: libdlib/package.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+<package format="2">
+	<name>libdlib</name>
+	<version>0.1.0</version>
+	<description>
+		This package wraps the external c++ library dlib (http://dlib.net/) in a ROS package, so other packages can use it. It downloads the source code of it and then builds it, corresponding to the web page of it, using cmake. The build library is a static library and will be copied to the created common/lib folder. The necessary header files will be copied to the common/include folder. The include folder needs to exist before building the package, because in the CMakeLists you have to specify where the headers are and if the folder doesn't exist, catkin won't be able to build the package. For further descriptions and tutorials see the Makefile.tarball and http://dlib.net/ .
+	</description>
+	<author email="florian.jordan@ipa.fraunhofer.de">Florian Jordan</author>
+	<maintainer email="richard.bormann@ipa.fraunhofer.de">Richard Bormann</maintainer>
+	<license>LGPL</license>
+	<!--url>http://ros.org/wiki/libdlib</url-->
```
Comment (floweisshardt) [inline libdlib/package.xml]: please create wiki entry or delete url.

File: libopengm/package.xml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+<package format="2">
+	<name>libopengm</name>
+	<version>0.1.0</version>
+	<description>
+		This package wraps the external c++ library opengm in a ROS package, so other packages can use it. It downloads the source code of it and then unzips it. The library is a header-only library with command line interfaces, which aren't used, so it doesn't gets build. For further descriptions and tutorials see the Makefile.tarball and https://github.com/opengm/opengm .
+	</description>
+	<author email="florian.jordan@ipa.fraunhofer.de">Florian Jordan</author>
+	<maintainer email="richard.bormann@ipa.fraunhofer.de">Richard Bormann</maintainer>
+	<license>LGPL</license>
+	<!--url>http://ros.org/wiki/libopengm</url-->
```
Comment (floweisshardt) [inline libopengm/package.xml]: please create wiki entry or delete url.

File: libconcorde_tsp_solver/package.xml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<package>
+	<name>libconcorde_tsp_solver</name>
+	<version>0.1.0</version>
+	<description>Wrapper for the concorde traveling salesman problem solver from http://www.math.uwaterloo.ca/tsp/concorde/downloads/downloads.htm</description>
+
+	<author email="florian.jordan@ipa.fraunhofer.de">Florian Jordan</author>
+	<maintainer email="rmb@ipa.fhg.de">Richard Bormann</maintainer>
```
Comment (floweisshardt) [inline libconcorde_tsp_solver/package.xml]: ...richard.bormann@ipa.fraunhofer.de...

File: libconcorde_tsp_solver/common/files/TSPlib_file.txt (hunk 1 above)
Comment (ipa-rmb) [inline libconcorde_tsp_solver/common/files/TSPlib_file.txt]: This was a necessary auto generated file, but we need write access - so now we create this file in ~/.ros.

**Comment by ipa-rmb** [comment]
All comments are now met

**Comment by floweisshardt** [comment]
CR: OK

**Pull request merged**